pkg/slo-controller/noderesource/noderesource.go

```diff
@@ -195,6 +199,88 @@ func (r *NodeResourceReconciler) updateGPUNodeResource(node *corev1.Node, device
 	return err
 }
 
+func (r *NodeResourceReconciler) updateGPUDriverAndModel(node *corev1.Node, device *schedulingv1alpha1.Device) error {
+	if device == nil {
+		return nil
+	}
+
+	hasGPUDevice := false
+	for _, device := range device.Spec.Devices {
+		if device.Type != schedulingv1alpha1.GPU {
+			continue
+		}
+		hasGPUDevice = true
+	}
+
+	if !hasGPUDevice {
+		return nil
+	}
+
+	modelList, driverVersion, err := getGPUModelAndDriverInfo()
```

Comment:
maybe this should be implemented in koordlet?

Reply:
+1

Reply:
The key question is whether we allow koordlet to modify Node. It seems that there are no features of koordlet to directly modify Node in the current code?

Reply:
fixed